src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -682,3 +682,23 @@ PARAM_DEFINE_INT32(MPC_OBS_AVOID, 0);
  * @group Mission
  */
 PARAM_DEFINE_INT32(MPC_YAW_MODE, 0);
+
+/**
+ * Flag to enable the use of a range sensor.
+ *
+ * @boolean
+ * @group Multicopter Position Control
+ */
+PARAM_DEFINE_INT32(MPC_USE_OBS_SENS, 0);
```

Comment:
You might want to move the OBS forward so that the relevant parameters are grouped together when viewed in the param tree.


    MPC_OBS_SENS_EN - enable use of obstacle avoidance sensors
    MPC_OBS_MIN_DIST - obstacle avoidance sensors min distance